packages/backend/src/peripherals/database/activity/BlockTransactionCountRepository.ts

```diff
@@ -43,7 +43,7 @@ export class BlockTransactionCountRepository extends BaseRepository {
 
   async deleteAll() {
     const knex = await this.knex()
-    return await knex('activity.block').delete()
+    return knex('activity.block').delete()
```

Comment:
why is there a need for this change?

Reply:
I tried to mimic the other repository tests ... That part was unnecessary so wanted to clean it

Reply:
but in other repositories we have the `await`

Reply:
I updated all 🙃